config/services.yaml

```diff
@@ -28,4 +28,8 @@ services:
     # as action arguments even if you don't extend any base controller class
     WMDE\BannerServer\Controller\:
         resource: '../src/Controller'
-        tags: ['controller.service_arguments']
\ No newline at end of file
+        tags: ['controller.service_arguments']
+
+    WMDE\BannerServer\Controller\BannerSelectionController:
+        arguments:
+            $bannerDirectory: 'banners/'
```

Comment:
I think this should be absolute, i.e. starting with `/`
Also, in this case it's not a "directory" but a "path", because it's part of a URL.